modules/graph/vertex_map/arrow_vertex_map_impl.h

```diff
@@ -41,28 +41,56 @@ void ArrowVertexMap<OID_T, VID_T>::Construct(const vineyard::ObjectMeta& meta) {
 
   this->fnum_ = meta.GetKeyValue<fid_t>("fnum");
   this->label_num_ = meta.GetKeyValue<label_id_t>("label_num");
+  this->use_perfect_hash_ = meta.GetKeyValue<bool>("use_perfect_hash_");
```

Comment:
Use `meta.GetKeyValue<bool>("use_perfect_hash_", this->use_perfect_hash_)` for backwards compatiblity (existing library or graph-loader or engine may create metadata that don't contains this key).

Reply:
Done.